user/pages/05.policy/03.admission/01.sigstore/docs.md

```diff
@@ -20,7 +20,7 @@ A private root of trust requires the the keys and/or certificates from your priv
 
 ![sigstore](2configure-private-root-of-trust.png)
 
-Next, for a given root of trust, configure each of the verifiers that you would like to use during admission control. There are two types of verifiers: keypair and keyless. A keypair verifier would be used when trying to verify an image signed by a user-defined private key. A keyless verifier would be used when verifying a signature generated by Sigstore's keyless pattern. More information about the Sigstore signing methods can be seen [here](https://docs.sigstore.dev/cosign/sign).
+Next, for a given root of trust, configure each of the verifiers that you would like to use during admission control. There are two types of verifiers: keypair and keyless. A keypair verifier would be used when trying to verify an image signed by a user-defined private key. A keyless verifier would be used when verifying a signature generated by Sigstore's keyless pattern. More information about the Sigstore signing methods can be seen [here](https://docs.sigstore.dev/cosign/overview/).
```

Comment:
This section is for signing & https://docs.sigstore.dev/cosign/sign is a valid link.
Why change it to https://docs.sigstore.dev/cosign/overview ?